playwright/reporter.js

```diff
@@ -1,4 +1,4 @@
-const stripAnsi = require('strip-ansi');
+import stripAnsi from 'strip-ansi';
```

Comment:
`strip-ansi` has moved to ESM: https://github.com/chalk/strip-ansi/commit/7cda68dcadde18b19bfa31b6223e9f0e60b3e319